src/kixi/lapita.clj

```diff
@@ -53,19 +53,18 @@
    (ds/dataset (csv-to-maps-coll filename)))
   ;; Returns a mix of coerced data and data inconsistent w/ schema
   ([filename schema]
-   (let [all-data (csv-to-maps-coll filename)
-         coerced-data (sc/coerce-data-from-schema all-data schema)]
-     (ds/dataset coerced-data)))
+   (csv-to-dataset filename schema {}))
   ;; Specify options in the 3rd arg to print number of rows with inconsistent data,
   ;; write inconsistent data to file return only coerced data.
-  ([filename schema {:keys [print-errors write-errors remove-errors]}]
+  ([filename schema {:keys [print-errors write-errors errors-filepath remove-errors]
+                     :or {errors-filepath "test-data/records-with-data-coercion-issues.csv"}}]
```

Comment:
This is still not a safe file path to assume. You either need to force them to specify a dir or create a temporary file using a safe method.